bears/typescript/TSLintBear.py

```diff
@@ -1,17 +1,33 @@
 import json
 
-from coalib.bearlib.abstractions.Lint import Lint
-from coalib.bears.LocalBear import LocalBear
-from coalib.misc.Shell import escape_path_argument
+from coalib.bearlib.abstractions.Linter import linter
 from coalib.results.Result import Result
 from coalib.settings.Setting import path
 
 
-class TSLintBear(LocalBear, Lint):
-    executable = 'tslint'
+@linter(executable='tslint')
+class TSLintBear:
+    """
+    Checks the code with ``tslint`` on each file separately.
+    """
+
+    @staticmethod
+    def create_arguments(filename, file, config_file,
+                         tslint_config: path = "",
+                         rules_dir: path = ""):
```

Comment:
We normally don't put spaces around the `=` here. Not sure if that's part of styleguide